### PR TITLE
Fix Table.show to not print enormous lines

### LINKED ISSUE #2405: Key table show should print wide tables in column groups, the way pandas does

### PR DESCRIPTION
**Pull request by tpoterba**
fixes #2405

### CODE REVIEW

**Comment by tpoterba** [comment]
Now prints like:
```text
+-----------------+-------------+-------------+---------------------------------+
| v               | rsid        |        qual | filters                         |
+-----------------+-------------+-------------+---------------------------------+
| Variant(GRCh37) | String      |     Float64 | Set[String]                     |
+-----------------+-------------+-------------+---------------------------------+
| 20:10019093:A:G | rs575534    | 1.68684e+06 | []                              |
| 20:10026348:A:G | NA          | 7.53522e+03 | []                              |
| 20:10026357:T:C | rs685723    | 3.75167e+05 | []                              |
| 20:10030188:T:A | rs652633    | 4.54459e+05 | []                              |
| 20:10030452:G:A | rs524625    | 4.30836e+05 | []                              |
| 20:10030508:T:C | rs118040899 | 9.27793e+03 | []                              |
| 20:10030573:G:A | rs139192723 | 7.19993e+03 | []                              |
| 20:10032413:T:G | rs683145    | 3.35656e+05 | []                              |
| 20:10036107:T:G | NA          | 1.94246e+03 | ["VQSRTrancheSNP99.95to100.00"] |
| 20:10036141:C:T | NA          | 1.97921e+03 | ["VQSRTrancheSNP99.80to99.90"]  |
+-----------------+-------------+-------------+---------------------------------+

+--------------------------+-------------+--------------+-----------------+----------+
| info.NEGATIVE_TRAIN_SITE |    info.HWP | info.AC      | info.culprit    | info.MQ0 |
+--------------------------+-------------+--------------+-----------------+----------+
| Boolean                  |     Float64 | Array[Int32] | String          |    Int32 |
+--------------------------+-------------+--------------+-----------------+----------+
| NA                       | 1.00000e-04 | [89]         | FS              |        0 |
| NA                       | 1.00000e+00 | [2]          | InbreedingCoeff |        0 |
| NA                       | 3.75700e-01 | [34]         | FS              |        0 |
| NA                       | 5.64000e-01 | [34]         | FS              |        0 |
| NA                       | 4.00300e-01 | [30]         | FS              |        0 |
| NA                       | 1.00000e+00 | [1]          | FS              |        0 |
| NA                       | 1.00000e+00 | [2]          | InbreedingCoeff |        0 |
| NA                       | 5.51300e-01 | [34]         | MQ              |        0 |
| NA                       | 8.28600e-01 | [13]         | FS              |        0 |
| true                     | 1.00000e+00 | [8]          | FS              |        0 |
+--------------------------+-------------+--------------+-----------------+----------+

+---------------------+---------+----------------------+----------------+----------------+
| info.ReadPosRankSum | info.AN | info.InbreedingCoeff | info.AF        | info.GQ_STDDEV |
+---------------------+---------+----------------------+----------------+----------------+
|             Float64 |   Int32 |              Float64 | Array[Float64] |        Float64 |
+---------------------+---------+----------------------+----------------+----------------+
|         3.53000e-01 |     200 |          1.35600e-01 | [0.582]        |    8.04930e+02 |
|         3.28000e-01 |     200 |         -5.20000e-03 | [0.005172]     |    7.07000e+01 |
|         3.03000e-01 |     200 |          3.26000e-02 | [0.23]         |    4.33130e+02 |
|         4.66000e-01 |     200 |          2.21000e-02 | [0.219]        |    5.36870e+02 |
|         4.43000e-01 |     200 |          3.12000e-02 | [0.216]        |    5.59990e+02 |
|         6.96000e-01 |     200 |         -2.90000e-03 | [0.002874]     |    1.13220e+02 |
|        -6.24000e-01 |     200 |         -2.90000e-03 | [0.002874]     |    1.02540e+02 |
|         2.91000e-01 |     200 |          2.27000e-02 | [0.221]        |    4.06470e+02 |
|         1.06000e-01 |     200 |         -8.20000e-02 | [0.032]        |    3.18100e+01 |
|        -2.19900e+00 |     200 |         -5.76000e-02 | [0.024]        |    3.08900e+01 |
+---------------------+---------+----------------------+----------------+----------------+
```

File: python/hail/api2/table.py [hunk 1]
```diff
@@ -1064,26 +1064,42 @@ def write(self, output, overwrite=False):
...
+        n : :obj:`int`
             Maximum number of rows to show.
-        truncate_to : bool
-            Truncate each column to the given number of characters
-        print_types : bool
+        width : :obj:`int`
+            Horizontal width at which to break columns.
+        truncate : :obj:`bool`, optional
+            Truncate each field to the given number of characters. I
+            ``None``, truncate fields to the given `width`.
```
Comment (patrick-schultz) [inline python/hail/api2/table.py]: `truncate` has type `int`. And I assume this should be "If ``None``...".

Comment (tpoterba) [reply python/hail/api2/table.py]: ah, good catch. bad copy/paste.

**Comment by konradjk** [comment]
Omg yas yas yas

**Pull request merged**